### ISSUE TITLE
cri-containerd tests not working due to docker-ce 18.06

### ISSUE THREAD

**Issue by chavafg** [kata-containers/tests#521]
```
1 attempt "sudo /home/zuul/bin/crictl --runtime-endpoint=unix:///run/containerd/containerd.sock info"! Trying again in 1 seconds...
=== RUN   TestImageLoad
E0720 13:04:46.245460   13132 remote_runtime.go:115] StopPodSandbox "287a38661b2769ea7d6105b9371f5a1751e23b302603b0b57897bcf6280998e6" from runtime service failed: rpc error: code = DeadlineExceeded desc = context deadline exceeded
E0720 13:04:46.246454   13132 remote_runtime.go:132] RemovePodSandbox "287a38661b2769ea7d6105b9371f5a1751e23b302603b0b57897bcf6280998e6" from runtime service failed: rpc error: code = Unknown desc = sandbox container "287a38661b2769ea7d6105b9371f5a1751e23b302603b0b57897bcf6280998e6" is not fully stopped
--- FAIL: TestImageLoad (60.96s)
        image_load_test.go:41: docker save image into tarball
        image_load_test.go:53: make sure no such image in cri
        image_load_test.go:60: load image in cri
        image_load_test.go:65: make sure image is loaded
        image_load_test.go:71: create a container with the loaded image
        image_load_test.go:89: make sure container is running
        Error Trace:    image_load_test.go:76
                        image_load_test.go:93
        Error:          Received unexpected error rpc error: code = DeadlineExceeded desc = context deadline exceeded

        Error Trace:    image_load_test.go:77
                        image_load_test.go:93
        Error:          Received unexpected error rpc error: code = Unknown desc = sandbox container "287a38661b2769ea7d6105b9371f5a1751e23b302603b0b57897bcf6280998e6" is not fully stopped

FAIL
./hack/test-utils.sh: line 53: 13031 Terminated              keepalive "sudo PATH=${PATH} ${ROOT}/_output/containerd ${CONTAINERD_FLAGS}" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log
Makefile:118: recipe for target 'test-integration' failed
make[1]: *** [test-integration] Error 1
make[1]: Leaving directory '/home/zuul/src/github.com/containerd/cri'
Makefile:54: recipe for target 'cri-containerd' failed
make: *** [cri-containerd] Error 2

```

**Comment by grahamwhaley** [comment]
Good find @chavafg - ouch :-( Yeah, we're going to have to find a nice way to handle that in the future.

**Comment by sboeuf** [comment]
@chavafg have you tried with a different port for CRIO and CRI-containerd ? Something like `10020` and `10030` ?

**Comment by sboeuf** [comment]
Here is the error we got:https://github.com/moby/moby/issues/37507

**Comment by sboeuf** [comment]
Two options for us, either we can configure containerd, or we change both CRI-O and CRI-containerd to use a different port. I would prefer change the docker port if possible.

**Comment by chavafg** [comment]
This was resolved on #530
